### PR TITLE
Agent 6

### PR DESCRIPTION
**Pull request by l0k0ms**
## Tl;Dr

Same idea as here: https://github.com/DataDog/documentation/pull/2060 and here: https://github.com/DataDog/documentation/pull/2071
Different layout

Agent 6 is coming: https://github.com/DataDog/datadog-agent
This PR updates the `/agent` section of the doc to document this new version.

LINK TO THE PREVIEW: https://docs-staging.datadoghq.com/gus/agent-6/agent/

### New layout
=== Basic agent usage section === 
[Link to the preview](https://docs-staging.datadoghq.com/gus/agent-6/agent/basic_agent_usage/)

The index now is a summary allowing to dive into the platforms list, with also a summary of configuration file location and links to agent configuration management tools setup.

All basic agent usage page now document:

* Agent v5 and v6 commands
* How to upgrade from agent v5 to v6
* How to downgrade from agent v6 to v5

=== Other Pages === 

* [Autodiscovery](https://docs-staging.datadoghq.com/gus/agent-6/agent/advanced_features/autodiscovery/)
* [Proxy](https://docs-staging.datadoghq.com/gus/agent-6/agent/advanced_features/proxy/)
* [Agent check](https://docs-staging.datadoghq.com/gus/agent-6/agent/advanced_features/agent_checks/)
* [Prometheus Check](https://docs-staging.datadoghq.com/gus/agent-6/agent/advanced_features/prometheus/)
* [Adding a custom python package to the agent](https://docs-staging.datadoghq.com/gus/agent-6/agent/advanced_features/custom_python_package/)

## Review 

* [ ] **Croissant** 
* [ ] **Kenafeh** 
* [x] **Ramen** 
* [ ] **Raclette**
* [ ] **Burrito** 

## To Do prior to merging

* [ ] merge https://github.com/DataDog/dogweb/pull/24458

### CODE REVIEW

File: content/agent/_index.md [hunk 1]
```diff
@@ -2,20 +2,29 @@
...
-{{< partial name="platforms/platforms.html" >}}
+<div class="alert alert-info">
+    Agent v6 is now available, <a href="https://github.com/DataDog/datadog-agent/blob/master/docs/agent/upgrade.md">upgrade to the newest version </a> to benefit from all new functionality 
+</div>
 
 ## What is the Agent?
 
 The Datadog Agent is a piece of software that runs on your hosts. Its job is to faithfully collect events and metrics and bring them to Datadog on
-your behalf so that you can do something useful with your monitoring and performance data.
+your behalf so that you can do something useful with your monitoring and performance data. The Datadog Agent is open-source, view the source code on GitHub for [Agent v5](https://github.com/DataDog/dd-agent) and [Agent v6](https://github.com/DataDog/datadog-agent). To see all changes between Agent v5 and v6, [consult our dedicated changes documentation](https://github.com/DataDog/datadog-agent/blob/master/docs/agent/changes.md).
```
Comment (irabinovitch) [inline content/agent/_index.md]: Open source shouldn't have a dash according to our style guide I believe.

File: content/agent/_index.md [hunk 2]
```diff
@@ -27,79 +36,168 @@ The Agent has three main parts: the collector, DogStatsD, and the forwarder:
...
 
-### Chef
-* [Chef Github project](https://github.com/DataDog/chef-datadog)
-* [Chef installation](https://app.datadoghq.com/account/settings#integrations/chef)
-* [Chef documentation](/integrations/chef)
+* Agent v6 has significantly improved resource usage over Agent v5:
+    
+    * It has decreased CPU usage:
+    * It has a decrease Memory usage:
+    * It uses fewer File Descriptors:
```
Comment (irabinovitch) [inline content/agent/_index.md]: should this be capitalized?

File: content/agent/_index.md [hunk 3]
```diff
@@ -27,79 +36,168 @@ The Agent has three main parts: the collector, DogStatsD, and the forwarder:
...
+    * It has an all around decreased footprint.
 
-### Puppet
-* [Puppet Github project](https://github.com/DataDog/puppet-datadog-agent)
-* [Puppet installation](https://app.datadoghq.com/account/settings#integrations/puppet)
-* [Puppet documentation](/integrations/puppet)
+* Global percentiles can be performed on the server directly to calculate real, effective global percentiles.
```
Comment (irabinovitch) [inline content/agent/_index.md]: Need to make it clear this is BETA and to email support for more information.

File: content/agent/_index.md (hunk 2 above)
Comment (jyee) [reply content/agent/_index.md]: Also: the memory line should probably be: `It uses less memory`
